### PR TITLE
Generate test fixtures with MessageFactory

### PR DESCRIPTION
**Pull request by maurodoglio**
This function will help to generate test data both for moztelemetry
 and telemetry-batch-view.

### CODE REVIEW

File: src/test/scala/com/mozilla/telemetry/heka/Resources.scala [hunk 1]
```diff
@@ -11,16 +11,33 @@ import java.io.ByteArrayOutputStream
...
+        Field(k, Some(Field.ValueType.BOOL), valueBool=Seq(v))
+      }
+      case (k: String, v: Double) => {
+        Field(k, Some(Field.ValueType.DOUBLE), valueDouble=Seq(v))
+      }
+      case (k: String, v: Long) => {
+        Field(k, Some(Field.ValueType.INTEGER), valueInteger=Seq(v))
+      }
+    }.toSeq
+    Message(ByteString.copyFromUtf8("1234"), 0, payload=Some("payload"), fields=fields)
```
Comment (vitillo) [inline src/test/scala/com/mozilla/telemetry/heka/Resources.scala]: Please make the `id` and the `payload` parametric.

File: src/test/scala/com/mozilla/telemetry/heka/Resources.scala [hunk 2]
```diff
@@ -11,16 +11,33 @@ import java.io.ByteArrayOutputStream
...
-  val message = {
-    val fields = List(
-      Field("bytes", Some(Field.ValueType.BYTES), valueBytes = Seq(ByteString.copyFromUtf8("foo"))),
-      Field("string", Some(Field.ValueType.STRING), valueString = Seq("foo")),
-      Field("bool", Some(Field.ValueType.BOOL), valueBool = Seq(true)),
-      Field("double", Some(Field.ValueType.DOUBLE), valueDouble = Seq(4.2)),
-      Field("integer", Some(Field.ValueType.INTEGER), valueInteger = Seq(42)))
-
-    Message(ByteString.copyFromUtf8("1234"), 0, payload = Some("payload"), fields = fields)
+  def messageFactory(fieldsMap: Map[String, Any]): Message = {
```
Comment (vitillo) [inline src/test/scala/com/mozilla/telemetry/heka/Resources.scala]: What about adding this method to the `Message` object using the "pimp my class pattern"?

**Comment by maurodoglio** [comment]
I added a companion object to RichMessage and I used its apply method to create Message objects.

**Comment by maurodoglio** [comment]
This is ready for another look 😺

**Pull request merged**